src/middlewared/middlewared/etc_files/local/smb4_share_load.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from middlewared.utils import run
+
+logger = logging.getLogger(__name__)
+
+
+async def reload_registry_shares(middleware):
+    drop = await run(['/usr/local/bin/net', 'conf', 'drop'], check=False)
```

Comment:
We're also working on Linux product now. On FreeBSD we have `/usr/local/bin` in `PATH` so just `net` will work and most likely it will also work in Linux from `/usr/bin`?